pkg/systemd/quadlet/quadlet.go

```diff
@@ -458,6 +460,11 @@ func ConvertContainer(container *parser.UnitFile, isUser bool) (*parser.UnitFile
 		podman.addf("--cap-add=%s", strings.ToLower(caps))
 	}
 
+	addSysctl := container.LookupAllStrv(ContainerGroup, KeyAddSysctl)
+	for _, sysctl := range addSysctl {
+		podman.addf("--sysctl=%s", strings.ToLower(sysctl))
```

Comment:
You should not add ToLower() here, there is at least one sysctl on my system which contains uppercase letters. And the values should not be converted either.

Reply:
Thanks for the comment. I haven't thought about the values.